src/main/kotlin/app/revanced/patches/reddit/ad/general/annotations/GeneralAdsCompatibility.kt

```diff
@@ -5,7 +5,7 @@ import app.revanced.patcher.annotation.Package
 
 @Compatibility(
     [Package(
-        "com.reddit.frontpage", arrayOf("2022.43.0")
+        "com.reddit.frontpage", arrayOf("2023.05.0")
```

Comment:
Why has the version be replaced instead of added?

Reply:
> 

To match with Layout Ads
Should I keep old version too?